addons/l10n_nl/migrations/12.0.3.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,47 @@
+---Fields in module 'l10n_nl'---
+---XML records in module 'l10n_nl'---
+DEL account.account.tag: l10n_nl.tag_nl_13 (noupdate)
+DEL account.account.tag: l10n_nl.tag_nl_34 (noupdate)
```

Comment:
You should manually try to delete these records (with the openupgradelib method)